topics/introduction/tutorials/galaxy-intro-101-everyone/workflows/main_workflow.ga

```diff
@@ -32,8 +32,8 @@
       "type": "data_input"
     },
     "1": {
-      "tool_id": "csv_to_tabular",
```

Comment:
@shiltemann I was testing all the workflows at this moment, and I came across some other imperfections, for example csv_to_tabular does not exist in the toolshed, and if I replace it with csv2tab_R, the inputs/outputs do not match anymore.

Reply:
that is probably one of the built-in tools in Galaxy (or a data type converter) so it should still work without installing anything

thanks for testing these!

Reply:
(@hexylena confirmed it is one of the datatype converters)

Reply:
Ok nice, than I will take away this change. It was the only one though were I guessed a tool manually. Weird thing is that my test instance with galaxy 19.01 does not recognize the tool.

Reply:
```
lib/galaxy/datatypes/converters/csv_to_tabular.xml
1:<tool id="csv_to_tabular" name="Convert CSV to tabular" version="1.0.0">
```

It was 19.05+

Reply:
explains everything, thank you !